diff -ru orig/vra4b37xql73282blh5kr3j6zzycc93i-mu-1.12.9-checkout/mu4e/mu4e-compose.el vra4b37xql73282blh5kr3j6zzycc93i-mu-1.12.9-checkout/mu4e/mu4e-compose.el
--- orig/vra4b37xql73282blh5kr3j6zzycc93i-mu-1.12.9-checkout/mu4e/mu4e-compose.el	2025-04-05 13:38:13.714178497 +0200
+++ vra4b37xql73282blh5kr3j6zzycc93i-mu-1.12.9-checkout/mu4e/mu4e-compose.el	2025-04-05 13:39:00.159231538 +0200
@@ -121,7 +121,7 @@
 Push mark at previous position, unless either a
 \\[universal-argument] prefix ARG is supplied, or Transient Mark mode
 is enabled and the mark is active."
-  (interactive "P")
+  (interactive "^P")
   (or arg
       (region-active-p)
       (push-mark))
@@ -138,7 +138,7 @@
 Push mark at previous position, unless either a
 \\[universal-argument] prefix ARG is supplied, or Transient Mark mode
 is enabled and the mark is active."
-  (interactive "P")
+  (interactive "^P")
   (or arg
       (region-active-p)
       (push-mark))
